tests/getById/endpoint.test.js

```diff
@@ -6,35 +6,149 @@ var config = require('../../config/load-config.js');
 
 var initServer = require('../../example/server.js');
 
+function endpoint (tag) {
```

Comment:
No change necessary, but the tests would read slightly clearer if this function was called 'getChallenge' or something like that.